### ISSUE TITLE
Email: Comments on WCAG 3.0 draft (6 - non-objective tests)

### ISSUE THREAD

**Issue by jspellman** [w3c/silver#485]
[Comment from Email](https://lists.w3.org/Archives/Public/public-silver/2021Mar/0002.html): 
from [Word docx](https://lists.w3.org/Archives/Public/public-silver/2021Mar/att-0002/Review_Comments_for_WCAG_3.0_-_Vanderheiden.docx) 

2)	Introduction of non-objective tests in WCAG-3.
In order for something to be a standard, particularly a standard that is going to be used in regulation of any type, 
•	all of the provisions that are normative (that is, the only ones you have to pay attention to in order to conform) must be objectively testable.  That is, one must either pass them or fail them.  
•	And you must have high inter-rater-reliability (that is, if you have a number of people who are aware of the technologies in use, a very high percentage of them would all come up with the same answer as to whether something passes or fails.  
•	If a number is used, (e.g. the volume must be above 3 dB), then the scale, the number that is the threshold, and the tool that can be used to measure the threshold value (number) that causes it to pass or fail, needs to be cited and the instrument creating the number needs to be objective.  
o	For example “The background noise must be 20 dB down from the foreground speech" would qualify.
	 but “The evaluator's opinion is that the score is a three or better" does not qualify since it relies on the opinion of an individual rather than on a measurement.  
o	Any measurements that rely on opinion are called qualitative rather than quantitative and are subjective rather than objective measures (unless one is measuring opinions rather than conformance).
This constrains the types of provisions or requirements that you can have in a standard.   Often leaving out guidance you would like to include but cannot reduce to an objectively testable requirement.
We ran into the same problem repeatedly in the WCAG 2 series. For example: we were able to require that the text alternative be available for any picture- since it is easy to objectively determine if a text alternative was present. But we could think of no objective measure that would allow us to require that the text alternative be of any particular quality.  We tried length and many other measures of alt text quality, and they all failed.  In the end we were only able to require that there was alt text for any (non-decorative) image.   We pushed it a bit in our “common failures” list for the provision by saying that the filename for the image was not sufficient if inserted as alt text.  But that was advisory not normative. 
Rationale / Why is it so important that everything be objective?  
First, it is unfair to an author to require that they do something, if they have no mechanism to determine if and when they have done enough.  For example, requiring that something have “sufficient contrast" is not good enough since an author would have no idea what contrast was sufficient.  Similarly requiring that they use clear language is not fair to the author unless it is objectively stated exactly what “clear language” is and exactly how to objectively measure it – along with a clear threshold for passing.  Clear to whom?  How clear?  How do I measure "clarity"? 
The second reason this is important is because regulations are completely useless if they cannot be enforced.  And the only way that a regulation can be enforced is in a court of law.  Regulations stem from laws and laws are enforced by the judicial system. (Occasionally there are other bodies designated such as the FCC or mediation bodies etc.  In all these cases someone still ends up "judging" whether or not the provision is met.)  If the provisions in a standard are not objective, the very first time it shows up in court, the defendants will cite, accurately, that the provision is not objective but rather is subjective.  And as a result, it is not enforceable. That is a real danger here.  The WCAG 2 series has so far not been challenged as being arbitrary.  The major problems we have had to date have been either around whether the law actually was meant to cover the web (which although it's been decided several times keeps being challenged) or around the cost for making things accessible (which we all know is higher if you build everything first and then try to retrofit accessibility into it).
It is doubtful that a nonobjective document in the WCAG series would be able to make it through to a “consensus document without objection” (as WCAG 2 was).  And if it was adopted over objections by the W3C, it is doubtful that international standards bodies would adopt it as they did the WCAG 2 series. Introducing something with the WCAG initials that is not objective would thus not only not achieve the objective of following along in the path of WCAG 2, but could hurt the WCAG line itself as well as cause confusion in the field.  There has already been pressure from and on other standards bodies to shift from WCAG and the W3C for setting web standards to ISO or other international standards bodies to create accessibility standards.  Up to the present it has been possible, based on the strength and quality of WCAG 2 series, and the backing of industry, to have the other international standards simply refer to WCAG for their web content accessibility provisions.  Creating a WCAG 3.0 that does not meet the requirements and standards of these standards bodies for provisions in their international standards could cause them to break from using the WCAG series in the future.  This would not be an arbitrary or capricious decision.  They simply would not be allowed, by the rules in their standards organizations, to include normative provisions that were not objective.   You will also note, that in their standard, they do not even allow three levels of conformance.  In their standards you either conform at level AA or you fail to conform.  For example, you do not have a choice in EN 301 549 to conform at either level A or AA.  Both A and AA provisions are included in their standard – and the requirement is that you conform at level AA or you don't conform.

**Comment by jspellman** [comment]
Thank you for your comment. Project members are working on your comment. You may see discussion in the comment thread and we may ask for additional information as we work on it. We will mark the official response when we are finished and close the issue.

**Comment by cwadamsoforacle** [comment]
Proposed response:

This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines (WCAG) 3.0](https://www.w3.org/TR/wcag-3.0/).  The Accessibility Guidelines Working Group attempted to address these concerns in this latest draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).
